instantsearch-compose/src/main/java/com/algolia/instantsearch/compose/filter/FacetListConnector.kt

```diff
@@ -1,25 +1,25 @@
 package com.algolia.instantsearch.compose.filter
 
 import com.algolia.instantsearch.compose.filter.internal.FacetListConnectionPager
-import com.algolia.instantsearch.compose.paging.SearcherPager
+import com.algolia.instantsearch.compose.paging.Paginator
 import com.algolia.instantsearch.core.connection.Connection
 import com.algolia.instantsearch.helper.filter.facet.FacetListConnector
 import com.algolia.instantsearch.helper.filter.facet.FacetListViewModel
 
 /**
  * Create a connection between a searcher pager and the facet list components.
  *
- * @param searcherPager pager to be connected
+ * @param paginator pager to be connected
  */
-public fun <T : Any> FacetListConnector.connectSearcherPager(searcherPager: SearcherPager<T>): Connection {
-    return viewModel.connectSearcherPager(searcherPager)
+public fun <T : Any> FacetListConnector.connectPaginator(paginator: Paginator<T>): Connection {
+    return viewModel.connectPaginator(paginator)
 }
 
 /**
  * Create a connection between a searcher pager and the facet list view model.
  *
- * @param searcherPager pager to be connected
```

Comment:
Doc update is required (not only here)

Reply:
Updated :)
Managed to move logic to internal implementation and reduce the public API complexity :)

Reply:
Well done 👍